c2cgeoform/views/abstract_views.py

```diff
@@ -63,6 +64,7 @@ def __init__(self,
         self._filter_column = filter_column if filter_column is not None \
             else self._attr if is_column \
             else None
+        self._visible = visible
```

Comment:
What is the purpose of this field ?

Reply:
A default configuration for columns visibility in grid